### ISSUE TITLE
Testing on Safari Desktop

### ISSUE THREAD

**Issue by janlelis** [palavatv/palava-web#15]
- [ ] Is palava.tv usable in general?
- [ ] Are there WebRTC connection bugs?
- [ ] Are there any unpleasant layout bugs?
- [ ] Any ideas for improvements on this platform?

**Comment by janlelis** [comment]
Blocked by #38

**Comment by pascalwengerter** [comment]
@janlelis just checked on Safarai (13.0.5) on macOS Catalina (10.15.3), seems to work fine 🙌

General improvement idea/suggestion: Provide an obvious (and easy to copy-paste) button/link to your room to invite others, e.g. in a Slack conversation. Copy-pasting the URL seems to work, but it's not _that_ easy to discover for first-time users 🙂

**Comment by janlelis** [comment]
@pascalwengerter Thank you for running the quick test and welcome to palava ;)

- Your comment spawned a new issue when to display the "copy link" button: https://github.com/palavatv/palava-web/issues/53 (copy link is also not 100% finished yet, since #23 is still missing)
- Besides this, I am happy to hear ideas about making the copy link feature more obvious (ux wise)
